packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -130,6 +130,15 @@ governing permissions and limitations under the License.
   box-sizing: border-box;
 
   outline: none; /* Hide focus outline around header */
+  & :first-child {
+    margin-top: 0;
+    padding-top: 0;
+  }
+
+  & :last-child {
+    margin-top: 0;
+    padding-bottom: 0;
+  }
```

Comment:
hmm... status light is merely an example of what someone might do, they could technically put in many things
I don't think this is the way we want to do it.
That said, maybe it's weird that these components have any sort of margin or padding by default? it prevents them from being aligned. I'm thinking we get rid of that.
If that's too invasive, then maybe this would be better? 
`.spectrum-Dialog-header > * {margin: 0, padding: 0}`

Reply:
Cool, that snippet works as well, thanks. I am personally still flip flopping on whether or not we should even do these overrides, kinda feel like we should just make the person using Dialog responsible for keeping things aligned within the `<Header>` component itself. Then the change in this pull would be just to add `UNSAFE_style:{{padding-top: 0}}` to the StatusLight in the story 

Open to opinions

Reply:
well we don't want people doing unsafe styles...

Reply:
yep agree 100%. Just keep thinking about if they have a custom Header component that has a bunch of stuff meticulously placed and then we come and wipe the padding/margins... (hence all the mental flip flopping that I'm doing right now 😅)